README.md

```diff
@@ -5,6 +5,34 @@ Once QIIME2 is [installed](https://docs.qiime2.org/2017.10/install/native/), and
     conda install -c https://conda.anaconda.org/biocore q2-fragment-insertion
     qiime dev refresh-cache
 
+## Why you should prefer fragment insertion aka SEPP over de-novo trees
+
+### Fragment insertion avoids artificially long outgroup branches that would lead to exaggerated separation in beta diversity.
+
+<img src="Example/denovoArtifacts.png">
+
+Beta diversity was computed for all 599 samples of this study (manuscript in preparation) on the non rarefied deblur table with 4,727 sOTUs total as unweighted unifrac distance with three alternative phylogenetic trees:
```

Comment:
It will be good to add the study ids and links to qiita here.

Reply:
This study is not yet public.

Reply:
That's fine, I _think_. You could add a note about this ...